resources/schemas/dbscripts/postgresql/cds-23.004-23.005.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE cds.sequence_header RENAME COLUMN source TO source_id;
+ALTER TABLE cds.antibody_sequence ADD COLUMN lineage BOOLEAN;
+ALTER TABLE cds.antibody_sequence DROP CONSTRAINT PK_cds_antibody_sequence;
+ALTER TABLE cds.antibody_sequence ADD CONSTRAINT PK_cds_antibody_sequence PRIMARY KEY (mab_id, sequence_id);
+
+ALTER TABLE cds.run_log RENAME TO alignment_run;
```

Comment:
Should we drop and re create instead, since we don't care about the log for previous runs? The PK_cds_run_log won't get renamed automatically, would it?

Reply:
Excellent point, now updated